apiserver/debuglog.go

```diff
@@ -40,20 +41,23 @@ type debugLogHandlerFunc func(
 	state.LogTailerState,
 	debugLogParams,
 	debugLogSocket,
+	string,
 	<-chan struct{},
```

Comment:
FWIW, I've previously done this elsewhere as a clean up and had this on my todo list but was looking to limit the scope of this PR. But I will do it here if you're happy to do a bigger PR :-)

Reply:
So it turns out this channel comes from
```
func (ctxt *httpContext) stop() <-chan struct{} {
	return ctxt.srv.tomb.Dying()
}
```

And is used in several places. I'd rather keep it as is and fix all of them in a separate PR, since the focus of this PR is wiring up the debug log reader and not fixing historical context issues.